tests/integration/tests/poi.js

```diff
@@ -178,6 +165,50 @@ test('display details about the poi on a poi click', async () => {
   expect(wiki_block).not.toBeFalsy()
 })
 
+async function select_poi_1(page) {
+  await page.evaluate(() => {
+    window.MAP_MOCK.evented.prepare('click', 'poi-level-1',  {originalEvent : {clientX : 1000},features : [{properties :{global_id : 1}}]})
+  })
+  await page.click('#mock_poi')
+  await wait(300)
+}
+
+test('add a poi as favorite and find it back in the favorite menu', async () => {
+  expect.assertions(7)
+  await page.goto(APP_URL)
+
+  // we select a poi and 'star' it
```

Comment:
Should be on an atomic test

Reply:
I don't really know how to do it since I want to test the favorite add and the favorite deletion.

Do you see an easy way to do it ?